File: Dev/C++/Mascaret/src/BEHAVE/OneShotBehaviorExecution.cpp
#include "BEHAVE/OneShotBehaviorExecution.h"

namespace BEHAVE
{

OneShotBehaviorExecution::OneShotBehaviorExecution(shared_ptr<Behavior> specif,shared_ptr<InstanceSpecification> host, const Parameters& p) : SimpleBehaviorExecution(specif,host,p)
{}

OneShotBehaviorExecution::~OneShotBehaviorExecution()
{}

bool OneShotBehaviorExecution::done()
{
	return true;
}

}

File: Dev/CS/Mascaret/Mascaret/BEHAVE/OneShotBehaviorExecution.cs
using System;
using System.Collections.Generic;

namespace Mascaret
{
    public class OneShotBehaviorExecution : SimpleBehaviorExecution
    {
        public OneShotBehaviorExecution(Behavior specif, InstanceSpecification host, Dictionary<String, ValueSpecification> p)
            : base(specif, host, p)
        {

        }

        public override bool done()
        {
            return true;
        }
    }
}

